src/core_plugins/getting_started/public/components/getting_started/getting_started.html

```diff
@@ -0,0 +1,135 @@
+<div class="kuiViewContent gettingStartedContent">
+
+  <div
+    ng-if="!gettingStarted.hasOptedOut"
+    class="kuiBar gettingStartedOptOutContent"
+  >
+    <div class="kuiBarSection">
+      <p class="kuiText kuiSubduedText">
+        <a
+          kbn-href="#/management"
```

Comment:
That was the direction I got from @skearns64 and @tbragin. I think the thinking is that users may want to opt out because they want to do other things in the Management area that are not index pattern creation. But I could be remembering that wrong -- Steve, Tanya?